app/src/main/AndroidManifest.xml

```diff
@@ -50,6 +50,8 @@
 
         <activity android:name=".activity.RightsActivity" />
 
+        <activity android:name=".activity.InfoActivity" />
```

Comment:
We probably need to add a theme here, right now I'm seeing a purpley statusbar which is ugly, and I think setting a theme should fix that.